mappings/net/minecraft/util/math/BlockBox.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_3341 net/minecraft/util/math/MutableIntBoundingBox
+CLASS net/minecraft/class_3341 net/minecraft/util/math/BlockBox
```

Comment:
We already have a class named `Box` which it's just an immutable version of this one (the same way we have `BlockPos` and `MutableBlockPos`). Either this class should be `MutableBox` and `Box` should stay `Box`, or this class should be `MutableBlockBox` and `Box` should be named `BlockBox`.

Reply:
The idea behind calling this one `BlockBox` is that it's the equivalent of the `Box` class but for blocks - it uses ints and is used only in block contexts. The main difference between the other `Box` and this one is not mutability, it's the type of data it contains.

Reply:
> The main difference between the other Box and this one is not mutability, it's the type of data it contains.

That's the problem. They contain exactly the same data (ints minX, minY, minZ, maxX, maxY, maxZ) but are named differently.

Reply:
Not quite, the original `Box` uses doubles, and is used in a lot of contexts. This one uses ints and is limited to block contexts.

Reply:
> the original Box uses doubles,

Sorry, I thought it was also using ints, I should have looked at the code.

Reply:
So... "conversation resolved"?